Source/Dafny/Compilers/Compiler-js.cs

```diff
@@ -2378,6 +2378,16 @@ protected class ClassWriter : IClassWriter {
       return SendToNewNodeProcess(dafnyProgramName, targetProgramText, callToMain, targetFilename, otherFileNames, outputWriter);
     }
 
+    DataReceivedEventHandler ProcessData(TextWriter writer) {
```

Comment:
I think you could provide a more descriptive name here. On the one hand it's actually a factory for event handlers, on the other hand the created event handler prints the data to the specified `writer`.